src/router/module.ts

```diff
@@ -50,7 +51,7 @@ mod.post('/install', async (req, res) => {
   try {
     res.json(await iasql.install(list, dbAlias, req.user));
   } catch (e: any) {
-    res.status(400).json(e.message);
+    res.status(500).json(logger.error(e));
```

Comment:
this is what it always should have been?

Reply:
400s are for user error, 500s are for server errors. It was a 400 here because failure to install was *expected* to always be because they're trying to install something that doesn't exist and not because installation itself fails.

Reply:
Well, it could be a bug on our end. I do get your point and will just leave as 500